CMakeLists.txt

```diff
@@ -139,11 +133,6 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # Define GNU standard installation directories.
 include(GNUInstallDirs)
 
-if(WIN32 AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-    # Windows: if install locations not set by user, set install prefix to "<build_dir>\install".
-    set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "default install path" FORCE)
-endif()
-
```

Comment:
This was removed from vvl/loader a bit ago. It's not needed and can cause users issues.

Reply:
What is the default without this?

Reply:
Not sure. The main issue is it prevents users/ci from setting [CMAKE_PREFIX_PATH](https://cmake.org/cmake/help/latest/envvar/CMAKE_PREFIX_PATH.html) as an environment variable.

Reply:
Also it breaks the new `--install` functionality which is also quite useful for package managers (and local users)

```
cmake --install build --prefix build/install
```